bin/pre-commit-hook.js

```diff
@@ -237,6 +249,8 @@ if ( phpFiles.length > 0 ) {
 		shell: true,
 		stdio: 'inherit',
 	} );
+
+	runPHPCSChanged( phpFiles );
```

Comment:
Why did you move this? It changes the order of things, in a way that makes things more confusing.

Moving it before line 257 means that things caught by PHPCompatibility will now be complained about twice.

Moving it before lines 260–279 will make it complain about some types of problems, then fix them, but still prevent the commit from proceeding.

Moving it before lines 281–298 will make it complain twice about some more things.

Reply:
I guess it is oversight from my side. I noticed that `runPHPCSChanged` is called even for commits without PHP files in it. It is fine, except for the fact that `runPHPCSChanged` calls `composer install`. So instead of adding conditional call (`if ( phpFiles.length > 0 )` ) I decided to move it into one of the similar blocks

Reply:
In c44d459c4fee88cfaea08f5121097879353f716f I reverted the order back